SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialAuthorityTest/SpatialAuthorityTest.cpp

```diff
@@ -29,13 +29,85 @@ ASpatialAuthorityTest::ASpatialAuthorityTest()
 	Description = TEXT("Test HasAuthority under multi-worker setups. It also ensures it works in Native");
 
 	Server1Position = FVector(-250.0f, -250.0f, 0.0f);
-	Server2Position = FVector(-250.0f, 250.0f, 0.0f);
+	Server2Position = FVector(250.0f, -250.0f, 0.0f);
+	Server3Position = FVector(-250.0f, 250.0f, 0.0f);
+	Server4Position = FVector(250.0f, 250.0f, 0.0f);
+	BorderPosition = FVector(0.0f, 0.0f, 0.0f);
 }
 
 void ASpatialAuthorityTest::PrepareTest()
 {
 	Super::PrepareTest();
 
+	FSpatialFunctionalTestStepDefinition NonReplicatedVerifyAuthorityStepDefinition(/*bIsNativeDefinition*/ true);
+	NonReplicatedVerifyAuthorityStepDefinition.StepName = TEXT("Non-replicated Dynamic Actor - Verify Authority  on Server 1");
+	NonReplicatedVerifyAuthorityStepDefinition.TimeLimit = 5.0f;
+	NonReplicatedVerifyAuthorityStepDefinition.NativeStartEvent.BindLambda([this]() {
+		// Not replicated so OnAuthorityGained() is not called.
+		if (VerifyTestActor(DynamicNonReplicatedActor, 1, 1, 0, 0))
+		{
+			FinishStep();
+		}
+	});
+
+	FSpatialFunctionalTestStepDefinition NonReplicatedVerifyOthersStepDefinition(/*bIsNativeDefinition*/ true);
```

Comment:
This name confused me for a little while - what about NonReplicatedVerifyNoInterestStepDefinition ?

Reply:
I am not sure if the name should include "no interest" as all the workers have interest at the border 0,0,0. Perhaps NonReplicatedVerifyNonAuthorityStepDefinition?

Reply:
Yes that's much better